### PR TITLE
wallet util cleanup and fixes for leveldb

### PR DESCRIPTION
**Pull request by HarshitJoshi**
cleanup and small fixes for wallet utils

### CODE REVIEW

**Comment by micahriggan** [comment]
Go ahead and add that logic to load wallet so you don't have to copy and paste this code

File: packages/bitcore-client/bin/wallet-tx [hunk 1]
```diff
@@ -9,22 +9,36 @@ program
...
-    'REQUIRED - Address of receiver, or json array [{address: abc1234, satoshis: xxxxx}]'
+    'REQUIRED - Address of receiver, or json array \'[{"address": "abc1234", "satoshis": xxxxx}]\''
   )
   .option('--amount <amount>', 'REQUIRED - Value to transfer')
   .option('--fee <fee>', 'REQUIRED - The transaction fee to pay')
   .option('--utxos <utxos>', 'REQUIRED - Unspent transactions that can be spent')
   .option('--change <change>', 'REQUIRED - Change addresses')
   .parse(process.argv);
 
+if (!program.path || !program.amount || !program.addresses || !program.fee || !program.utxos || !program.change) {
```
Comment (micahriggan) [inline packages/bitcore-client/bin/wallet-tx]: What if we made a thing that did something like 

```
for(option in program.options) {
  if(option.required && !program.option){
   console.error(`${option.name} is required to run this command`)
  }
}
```

Comment (micahriggan) [reply packages/bitcore-client/bin/wallet-tx]: Then we could just make our own little wrapper around commander that did that

File: packages/bitcore-client/bin/wallet-register [hunk 2]
```diff
@@ -10,12 +10,26 @@ program
...
+}
+
 const main = async () => {
   const { path, baseUrl } = program;
   let wallet;
   try {
     const password = await promptly.password('Wallet Password:');
+    if (!password) {
+      throw new Error(
+        'No password provided. Keys can\'t be imported without unlocking the wallet'
```
Comment (micahriggan) [inline packages/bitcore-client/bin/wallet-register]: I think the password can be empty, let's see.

File: packages/bitcore-client/lib/wallet.js [hunk 3]
```diff
@@ -53,6 +56,13 @@ class Wallet {
 
   static async loadWallet(params) {
     const { path } = params;
+    try {
+      await accessAsync(path, fs.constants.F_OK | fs.constants.R_OK);
+      await accessAsync(path + '/LOCK' || path + 'LOCK', fs.constants.F_OK | fs.constants.R_OK | fs.constants.W_OK);
+      await accessAsync(path + '/LOG' || path + 'LOG', fs.constants.F_OK | fs.constants.R_OK | fs.constants.W_OK);
+    } catch (err) {
+      return console.error('Invalid wallet path');
```
[the remark this answers is not in the document]
Comment (HarshitJoshi) [reply packages/bitcore-client/lib/wallet.js]: resolved

File: packages/bitcore-client/bin/wallet-utxos [hunk 4]
```diff
@@ -9,12 +9,26 @@ program
...
+}
+
 const main = async () => {
   const { path } = program;
   let wallet;
   try {
     const password = await promptly.password('Wallet Password:');
+    if (!password) {
+      throw new Error(
+        'No password provided. Keys can\'t be imported without unlocking the wallet'
```
[the remark this answers is not in the document]
Comment (HarshitJoshi) [reply packages/bitcore-client/bin/wallet-utxos]: updated

File: packages/bitcore-client/bin/wallet-register (hunk 2 above)
Comment (HarshitJoshi) [reply packages/bitcore-client/bin/wallet-register]: updated

File: packages/bitcore-client/bin/wallet-tx (hunk 1 above)
Comment (HarshitJoshi) [reply packages/bitcore-client/bin/wallet-tx]: #1618

**Pull request merged**